scrapy/pqueues.py

```diff
@@ -130,9 +111,16 @@ def _active_downloads(self, slot):
 
 
 class DownloaderAwarePriorityQueue:
-    """ PriorityQueue which takes Downloader activity into account:
-    domains (slots) with the least amount of active downloads are dequeued
-    first.
+    """:ref:`Priority queue <priority-queues>` that keeps a similar number of
+    on-going requests per domain.
```

Comment:
I think that'd be nice to keep mentioning slots here, it is not necessarily per domain.